manifests/t/TUitTUWien/TUtoolbox/2023.0.0.666/TUitTUWien.TUtoolbox.locale.de-DE.yaml

```diff
@@ -0,0 +1,24 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.locale.1.2.0.schema.json
+
+PackageIdentifier: TUitTUWien.TUtoolbox
+PackageVersion: 2023.0.0.666
+PackageLocale: de-DE
+Publisher: TU Wien - TU.it
+PublisherUrl: https://www.it.tuwien.ac.at/
+PublisherSupportUrl: https://www.it.tuwien.ac.at/services/beratung-und-servicedesk/servicedesk/service-center
+PrivacyUrl: https://www.tuwien.at/index.php?eID=dms&s=4&path=Dokumente/Datenschutzerkl%C3%A4rungen%20Sonstige/Datenschutzerkl%C3%A4rung_Websites.pdf
+Author: Christian Beck
+PackageName: TUtoolbox
+PackageUrl: https://service.it.tuwien.ac.at/swsupport/tools/tutoolbox
+License: Proprietäre Software von TU.it, TU Wien
+# LicenseUrl: 
+Copyright: © 2021-2023 TU.it, TU Wien
+# CopyrightUrl: 
+ShortDescription: Ein Programm zur Installation aller benötigten Werkzeuge und Patches für TU-Mitglieder.
+ReleaseNotesUrl: https://service.it.tuwien.ac.at/swsupport/tools/tutoolbox/index.html#history
+Moniker: tutoolbox
```

Comment:
```suggestion
```
Only the `ManifestType: defaultLocale` supports Moniker field

Reply:
changed it with the current push